examples/create_block_mesh.py

```diff
@@ -3,16 +3,18 @@
 
```

Comment:
Would it not make more sense to call this file create_quad_mesh.py? Or quad or something like this since block-mesh is as far as I know not a term for quadrilateral meshes?

Reply:
Could we maybe also add more descriptions to the example files? These are intended to bring the concepts and function of the parts to new users. So in this case more comments describing, that we are here creating a mesh of a square of size 2x2 and mesh it with 2 elements (3 nodes) in the x-direction and 3 elements (4 nodes) in the y-direction? I know that these can be looked up in the function documentation but it might help with bringing new users faster up to speed. And it is also much easier to understand if one is not currently in an IDE where the function lookup is instant.  

Just a general thought @j042 ?

Reply:
1 -> yes!
2 -> sounds great.